[docs] Readme updates for PyTorch 1.11 (#1709)
deepjavalibrary/djl: android/pytorch-native/README.md

# Pytorch Android Build Procedure

> Follow this setup guide in order to run DJL apps on an Android. In order to successfully run the apps, you must install several Android items and build Android ABIs outlined below.

Reviewer: We should make it clear this is not for regular user, this is only for developers who want to build pytorch naitve binaries from source.

## Prerequisites

```

> # Run the following command if you have python3 installed

Reviewer: ```suggestion
# Run the following command (assume you have python3 installed already)
```

export PYTHON=python3

# install needed python libraries

> pip3 install numpy ninja pyyaml mkl mkl-include setuptools cmake cffi typing_extensions

Reviewer: ```suggestion
python3 -m pip install numpy ninja pyyaml mkl mkl-include setuptools cmake cffi typing_extensions
```

```
## Only run the android-sdk install specific to your OS, then proceed to installing command line tools below

This will install the android-sdk on your machine as well as python3. It sets the ANDROID_HOME environment variable for use throughout this setup guide, and gives you ownership to the ANDROID_HOME directory which we need later.

### Linux (Ubuntu 20.04) android-sdk install

```
# install python and Android sdk
sudo apt-get install android-sdk python3

# set Android home
export ANDROID_HOME=/usr/lib/android-sdk

# give user ownership of ANDROID_HOME path
sudo chown -R ubuntu:ubuntu $ANDROID_HOME
```

### Mac android-sdk install

```
# install python and Android sdk

> brew install android-sdk python3

Reviewer: ```suggestion
brew install android-sdk
```
